app/services/hearings_creator.rb

```diff
@@ -51,17 +55,41 @@ def push_appeals
                   appeal_data: {
                     appeal_type: appeal.dig(:type, :applicationCode),
                     appeal_outcome: appeal[:applicationOutcomes],
-                  })
+                  },
+                  application_data: nil,
+                  function_type: nil)
     end
   end
 
-  def push_to_sqs(shared_time:, case_urn:, defendant:, appeal_data:)
+  def push_applications
+    hearing[:courtApplications]&.each do |application|
+      defendant_cases = application.dig(:applicant, :masterDefendant, :defendantCase)
+      return if defendant_cases.nil?
+
+      defendant_cases.each do |defendant_case|
+        defendant = LaaReference.find_by(defendant_id: defendant_case[:defendantId], linked: true)
+        next if defendant.nil?
+
+        push_to_sqs(shared_time: shared_time,
+                    case_urn: application[:applicationReference],
```

Comment:
After a talk with Ahilan, this should be accessed via the CDA database. But might be worth checking if it's present anywhere in the court_application data for us to pluck out.

Reply:
I don't actually know what we want to send here to know what to look for in court_application